sagemaker-featurestore/feature_store_object_detection_ground_truth.ipynb

```diff
@@ -0,0 +1,484 @@
+    "\n",
+    "### Overview\n",
+    "1. Set up\n",
+    "2. Prepare `output.manifest`for Feature Store. \n",
+    "3. Create a feature group and ingest your data into it.\n",
+    "\n",
+    "### Prerequisites\n",
+    "This notebook uses the Python SDK library for Feature Store, and the `Python 3 (Data Science)` kernel. To encrypt your data with KMS key for server side encryption, you will need to have an active KMS key. If you do not have a KMS key, then you can create one by following the [KMS Policy Template](https://sagemaker-examples.readthedocs.io/en/latest/sagemaker-featurestore/feature_store_kms_key_encryption.html#KMS-Policy-Template) steps or you can visit the [KMS section in the console](https://console.aws.amazon.com/kms/home) and follow the prompts for creating a KMS key. This notebook is compatible with SageMaker Studio, Jupyter, and JupyterLab. \n",
+    "\n",
+    "### Library Dependencies:\n",
+    "* `sagemaker>=2.0.0`\n",
+    "* `numpy`\n",
+    "* `pandas`\n",
+    "* `boto3`\n",
+    "\n",
+    "### Data\n",
+    "This notebook uses a synthetic manifest file. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import sagemaker\n",
+    "import os"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Set up"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sagemaker_session = sagemaker.Session()\n",
+    "s3_bucket_name = sagemaker_session.default_bucket() # This is the bucket for your Feature Store.\n",
+    "prefix = 'sagemaker-featurestore-demo'\n",
+    "role = sagemaker.get_execution_role()\n",
+    "region = sagemaker_session.boto_region_name"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Additional - Helper Method\n",
+    "Below is a method that you can use to get your manifest file from your S3 bucket into your current working directory."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def download_file_from_s3(bucket, path, filename):\n",
+    "    '''\n",
+    "    bucket: S3 bucket name\n",
+    "    path: path to file\n",
+    "    filename: the name of the file you are downloading\n",
+    "\n",
+    "    Download filename to your current directory.\n",
+    "    Return: None\n",
+    "    '''\n",
+    "    import os.path\n",
+    "    if not os.path.exists(filename):\n",
+    "        s3 = boto3.client('s3')\n",
+    "        s3.download_file(\n",
+    "            Bucket = bucket,\n",
+    "            Key = path,\n",
+    "            Filename = filename\n",
+    "        )\n",
+    "# Supply the path to your output.manifest file from your AWS Ground Truth Classification labelling job. \n",
```

Comment:
"# Supply the path to your `output.manifest` file from your Amazon SageMaker Ground Truth classification labeling job. \n",